### DISCUSSION TITLE
[@types/bootstrap-select] BootstrapVersion property is not available

### DISCUSSION THREAD

**Discussion by dehy** [DefinitelyTyped/DefinitelyTyped#55073] [Issues with a @types package]
Hello there 👋 (oh, hello Github Discussions)

I'm using a special version of Bootstrap v4 that makes the bootstrap-select's BootstrapVersion auto-discovery non available. [Official Docs](https://developer.snapappointments.com/bootstrap-select/options/) says that we can manually override the version used by setting `$.fn.selectpicker.Constructor.BootstrapVersion`.

Unfortunately, tsc complains about `Constructor`:

My line:
```javascript
$.fn.selectpicker.Constructor.BootstrapVersion = '4';
```

The error:
```
[tsl] ERROR in /path/to/app/assets/js/app.ts(20,19)
      TS2339: Property 'Constructor' does not exist on type '{ (opts?: BootstrapSelectOptions | undefined):JQuery<HTMLElement>; (method: MethodType, ...args: (string | string[])[]): JQuery<...>; }'.
```

I'm not expert enough to understand what to update in the definition file.

Versions:
- tsc: 4.3.5
- webpack: 4.43.0

Ping Authors: @LKay @AlexTruba 

Thank you for your work!

**Comment by dehy** [comment]
Ok, I've just found a way to make it work by using "[Call Signatures](https://www.typescriptlang.org/docs/handbook/2/functions.html#call-signatures)"

```js
...

interface JQuery {
    selectpicker: SelectPicker;
}

type SelectPicker = {
    Constructor: { BootstrapVersion: string }
    (opts?: BootstrapSelectOptions): JQuery;
    (method: MethodType, ...args: Array<string | Array<string>>): JQuery;
}
```

Is that a valid solution for you? I can make a PR if you want

**Reply by AlexTruba** [reply]
Hi @dehy. Yes, it's a valid solution and good for me. 
But there is already created a PR with these changes #55075. 
Please follow it in the scope of this issue.
